bin/segment2.c

```diff
@@ -2968,7 +2969,8 @@ const Gfx dl_ia8_up_arrow_end[] = {
     gsSPEndDisplayList(),
 };
 
-// 0x02014958 - 0x02014960
+#include "segment2/controllers.c.in"
+#include "segment2/rumble_pak.c.in"
```

Comment:
these should be .inc.c files for consistency with the repo. i can help you create a makefile rule to exclude them from the C file list if you like (this is something that should be done anyway tbh)